### DISCUSSION TITLE
PHPUnit test ?

### DISCUSSION THREAD

**Discussion by opencart-course** [opencart/opencart#10851] [Show and tell]
Tell me please
Is there any interest in the future to cover the code base with tests?
For example, PHPUnit ?

**Comment by anytizer** [comment]
Unit testing OpenCart itself in developer's perspective (code testing) may be a core story that you are asking for.

But I have also started writing some PHPUnit snippets for merchant's perspectives - at [OCUnit](https://github.com/anytizer/ocunit). Please have a look.
